packages/quantic/force-app/main/default/lwc/quanticCaseClassification/quanticCaseClassification.js

```diff
@@ -0,0 +1,285 @@
+  }
+
+  /**
+   * Shows an error message in the component if there is an error.
+   * @returns {void}
+   */
+  @api reportValidity() {
+    if (this.required && !this._value) {
+      this._errorMessage = this.messageWhenValueMissing;
+    } else {
+      this._errorMessage = '';
+    }
+  }
+
+  /**
+   * Returns the value of the case classification input.
+   * @returns {string}
+   */
+  @api get value() {
+    return this._value;
+  }
+
+  /**
+   * Returns the error message to be shown.
+   * @type {string}
+   */
+  get errorMessage() {
+    return this._errorMessage;
+  }
+
+  /**
+   * Tells if there is more options to show in addition to the suggested options.
+   * @returns {boolean}
+   */
+  get isMoreOptionsVisible() {
+    return this.options.length > Math.max(Number(this.maxChoices), 1);
+  }
+
+  /**
+   * Tells if the select input is visible.
+   * @type {boolean}
+   */
+  get isSelectVisible() {
+    return this._isSelectVisible || 0 >= this.suggestions.length;
+  }
+
+  /**
+   * Tells if the suggested options are visible.
+   * @type{boolean}
+   */
+  get isSuggestionsVisible() {
+    return this._isSuggestionsVisible;
+  }
+
+  /**
+   * Returns the CSS class of the form.
+   * @returns {string}
+   */
+  get formClass() {
+    return `slds-form-element ${this.hasError ? 'slds-has-error' : ''}`;
+  }
+
+  /**
+   * Returns the options, excluding the suggestions.
+   * @returns {Array<object>}
+   */
+  get filteredOptions() {
+    return this.options.filter((option) => 
+      !this.classifications.some((item) => item.value === option.value)
+    );
+  }
+
+  /**
+   * Returns the suggestions to be shown.
+   * @returns {Array}
+   */
+  get suggestions() {
+    return this.classifications.slice(
+      0,
+      Math.max(Number(this.maxChoices), 1) - 1
+    );
+  }
+
+  /**
+   * Shows the select input.
+   * @returns {void}
+   */
+  showSelect() {
+    this._isSelectVisible = true;
+  }
+
+  /**
+   * Handles the selection of a suggestion.
+   * @returns {void}
+   */
+  handleSelectClassification(event) {
+    const suggestionId = event.target.dataset.suggestionId;
+    const value = event.target.value;
+    this.engine.dispatch(this.actions.logClassificationClick(suggestionId));
+    this.field.update(value);
+    this._errorMessage = '';
+    this._value = value;
+  }
+
+  /**
+   * Handles the select input change.
+   * @returns {void}
+   */
+  handleSelectChange(event) {
+    const value = event.target.value;
+    this.field.update(value);
+    this._errorMessage = '';
+    this._value = value;
+    if (this._isSuggestionsVisible && this.isMoreOptionsVisible) {
+      this._hideSuggestions();
+    }
+  }
+
+  /**
+   * Hide the suggested options.
+   * @returns {void}
+   */
+  _hideSuggestions() {
+    const suggestions = this.template.querySelectorAll('.slds-visual-picker');
+    suggestions.forEach((suggestion) => {
+      // @ts-ignore
+      suggestion.style.width = `${suggestion.clientWidth}px`;
```

Comment:
Yes It would be simpler to perform the hiding using the html conditional templates  and without the animation effect. The UX team preferred to have this animation  effect so that's why it's here,  is there any reason we want to avoid having animations effects? or is it a question of having this component consistant with the other quantic components, like I understand that having all the other Quantic components without any animation effects and this one containing one would be a bit weird.